sass/themes/cr/2019/components/membership-signup/js/membership-signup.js

```diff
@@ -165,8 +165,7 @@
 			});
 
 			/* Handle case where users are taken back to cr.com from donation */
-			if( url_string.indexOf('&amount=') > -1 && amountValue.length > 0 ) {
-				if(rowIDValue.length > 0) {
+			if( url_string.indexOf('&amount=') > -1 && amountValue.length > 0 && rowIDValue.length && url_string.indexOf('&rowID=') > -1  ) {
 					$("#" + rowIDValue).find('.select-amount-btn').each(function (i) {
 						var currentMoneyBuyValue = getMoneyBuyValue($(this));
```

Comment:
Doesn't look like we need this currentMoneyBuyValue var at all here?

Reply:
We need it to check if it matches the value of amount params so that we can highlight either one of the money buy or the input field

Reply:
Sorry, I totally misread this; you're using it on the next line 😆